integr8ly/integreatly-operator
# INTLY-4661 - update code ready operator to use 2.0.0

## Description
Upgrade the code ready operator version to 2.0.0 GA

Jira
* https://issues.redhat.com/browse/INTLY-4661

## Verification
* Clone this branch
* Run the operator locally against cluster with at least these products installed:
```
operator-sdk up local --namespace=integreatly --operator-flags=--products=rhsso,rhssouser,cloud-resources,codeready-workspaces
```
* Ensure codeready gets installed correctly
* Run the `setup-htpass-idp.sh` script and note the outputed password for `customer-admin`
```
./scripts/setup-htpass-idp.sh
```
* Login in to code ready using the Single Sign On option using the `htpasswd` provider using the `customer-admin` credentials as noted above
* Input account information while keeping username as `customer-admin`
* Link to existing `customer-admin` accont, using the password: `Password1`
* Ensure user signed it correctly and can provision a workspace without visible errors
